libraries/AP_Perf/AP_Perf_Linux.cpp

```diff
@@ -92,7 +86,7 @@ AP_Perf_Linux::AP_Perf_Linux()
     _perf_counters.reserve(50);
 
 #ifdef DEBUG_PERF
-    hal.scheduler->register_timer_process(FUNCTOR_BIND_MEMBER(&AP_Perf_Lttng::_debug_counters, void));
+    hal.scheduler->register_timer_process(FUNCTOR_BIND_MEMBER(&AP_Perf_Linux::_debug_counters, void));
```

Comment:
ha corrected here!

Reply:
Yep, I took original commits as is and added changes on top, including this